### ISSUE TITLE
cannot select ch 123 left/right

### ISSUE THREAD

**Issue by xgui3783** [FZJ-INM1-BDA/siibra-python#30]
version/hash: 6831d2df53fe5715738ccb752604fcdfabc32340
to reproduce:

```python
import siibra as sb
atlas=sb.atlases.MULTILEVEL_HUMAN_ATLAS
atlas.select_region('123 left')
[siibra:ERROR]  Cannot select region. The spec "123 left" is not unique. It matches: Ch 123 (Basal Forebrain) - left hemisphere, Ch 123 (Basal Forebrain) - left hemisphere
```

**Comment by dickscheid** [comment]
this was a bug that had been fixed already on dev. Can you test again?

**Comment by xgui3783** [comment]
issue sitll exists, at hash e155a16beebdef9b5ca0219168ba0a0c7b255a8e

**Comment by dickscheid** [comment]
still persists in 0.1a4.

**Comment by dickscheid** [comment]
Fixed in b0a1fd7 , now the list of found regions is unique.
